src/SectionPostListing.js

```diff
@@ -61,7 +61,13 @@ const SectionPostListing = React.createClass({
 
   getDefaultProps: function() {
     return {
-      postsTransform: _.identity,
+      postsTransform: (articles) => {
+        return _.chain(articles)
+          .sortBy('published')
+          .uniq('url')
```

Comment:
Saw some repeated articles so this is necessary...should look into the root cause though.